tests/integration/test_get_records.py

```diff
@@ -317,3 +328,150 @@ def test_get_record_when_user_is_not_member_of_collaboration_returns_403(
+    deposit.publish()
+    _, rec = deposit.fetch_published()
+
+    pid = deposit['control_number']
+    headers = auth_headers_for_example_user + [('Accept', 'application/form+json')]
+
+    resp = client.get(f'/records/{pid}', headers=headers)
+    assert resp.status_code == 200
+    assert resp.json == {
+        'access': {
+            'record-admin': {'roles': [], 'users': [example_user.email]},
+            'record-read': {'roles': [], 'users': [example_user.email]},
+            'record-update': {'roles': [], 'users': [example_user.email]}},
+        'can_update': True, 'is_owner': True,
+        'created': rec.created.strftime('%Y-%m-%dT%H:%M:%S.%f+00:00'),
+        'updated': rec.updated.strftime('%Y-%m-%dT%H:%M:%S.%f+00:00'),
+        'created_by': example_user.email,
+        'experiment': rec['_experiment'],
+        'draft_id': rec['_deposit']['id'],
+        'id': pid,
+        'files': [], 'labels': [],
+        'links': {
+            'html': f'http://analysispreservation.cern.ch/published/{pid}',
+            'self': f'http://analysispreservation.cern.ch/api/records/{pid}'
+        },
+        'metadata': {}, 'revision': 0, 'status': 'published', 'type': 'record',
+        'schema': {'name': 'cms', 'version': '1.0.0'},
+        'schemas': {'schema': {'title': {'type': 'string'}}, 'uiSchema': {}}
+    }
+
+
+def test_get_record_with_form_json_serializer_with_schema(
+        client, auth_headers_for_example_user, example_user, create_deposit, create_schema):
+    create_schema(
+        'test-schema',
+        experiment='CMS',
+        deposit_schema={
+            'title': 'record-test-schema', 'type': 'object',
+            'properties': {
+                'title': {'type': 'string'},
+                'date': {'type': 'string'}
+            }
+        },
+        deposit_options={
+            'title': 'ui-test-schema', 'type': 'object',
+            'properties': {
+                'title': {'type': 'string'},
+                'field': {'type': 'string'}
+            }
+        }
+    )
+    deposit = create_deposit(example_user, 'test-schema')
+    deposit.publish()
+    _, rec = deposit.fetch_published()
+
+    pid = deposit['control_number']
+    headers = auth_headers_for_example_user + [('Accept', 'application/form+json')]
+    resp = client.get(f'/records/{pid}', headers=headers)
+
+    assert resp.status_code == 200
+    assert resp.json['schema'] == {
+        'name': 'test-schema',
+        'version': '1.0.0'
+    }
+    assert resp.json['schemas']['schema'] == {
+        'properties': {
+            'date': {'type': 'string'},
+            'title': {'type': 'string'}
+        },
+        'title': 'record-test-schema',
+        'type': 'object'
+    }
+    assert resp.json['schemas']['uiSchema'] == {
+        'properties': {
+            'field': {'type': 'string'},
+            'title': {'type': 'string'}
+        },
+        'title': 'ui-test-schema',
+        'type': 'object'
+    }
```

Comment:
You can merge those tests in one, plus check metadata and files fields, sth like that:
```
def test_get_record_when_form_json_serializer_returns_correct_fields(client, example_user,
                                                create_schema, create_deposit,
                                                auth_headers_for_example_user):

    create_schema('test-schema',
                  experiment='CMS',
                  deposit_schema={'title': 'deposit-test-schema'},
                  deposit_options={'title': 'ui-test-schema'},
                  record_schema={'title': 'record-test-schema'},
                  record_options={'title': 'record-ui-test-schema'},
                  use_deposit_as_record=False)  # thats how you can use it, if you want to - you dont want to serve deposit schemas as record ones, you want to define your own record schemas, different than deposit ones
    deposit = create_deposit(example_user,
                             'test-schema', 
                             {  # forgot to test metadata
                                 '$ana_type': 'test-schema',
                                 'my_field': 'mydata'
                             },
                             experiment='CMS',
                             files={'readme': BytesIO(b'Hello!')},  # thats how you can add file 
                             publish=True)  # that how you can publish your record
    pid = deposit['control_number']
    _, record = deposit.fetch_published()
    metadata = record.get_record_metadata()
    file = deposit.files['readme']

    resp = client.get(f'/records/{pid}',
                      headers=auth_headers_for_example_user +
                      [('Accept', 'application/form+json')])

    assert resp.status_code == 200
    assert resp.json == {
        'access': {
            'record-admin': {
                'roles': [],
                'users': ['cms_user@cern.ch']
            },
            'record-read': {
                'roles': [],
                'users': ['cms_user@cern.ch', 'cms_user2@cern.ch']
            },
            'record-update': {
                'roles': [],
                'users': ['cms_user@cern.ch']
            }
        },
        'created_by': example_user.email,
        'is_owner': True,
        'can_update': True,
        'experiment': 'CMS',
        'created': metadata.created.strftime('%Y-%m-%dT%H:%M:%S.%f+00:00'),
        'updated': metadata.updated.strftime('%Y-%m-%dT%H:%M:%S.%f+00:00'),
        'id': pid,
        # TOFIX shouldnt be deposit.files but record.files here!  (this is not your issue, leave tofix for later)
        'files': [{
            'bucket': str(deposit.files.bucket),
            'checksum': file.file.checksum,
            'key': file.key,
            'size': file.file.size,
            'version_id': str(file.version_id)
        }],
        'links': {
            'bucket': 'http://analysispreservation.cern.ch/api/files/{}'.
            format(record.files.bucket),
            'html': 'http://analysispreservation.cern.ch/published/{}'.format(
                pid),
            'self': 'http://analysispreservation.cern.ch/api/records/{}'.
            format(pid),
        },
        'draft_id': deposit.pid.pid_value,
        'metadata': {
            'my_field': 'mydata'
        },
        'revision': 0,
        'status': 'published',
        'type': 'record',
        'labels': [],
        'schema': {
            'name': 'test-schema',
            'version': '1.0.0'
        },
        'schemas': {
            'schema': {
                'title': 'record-test-schema',
            },
            'uiSchema': {
                'title': 'record-ui-test-schema',
            }
        },
    }
```